pkgs/applications/misc/xpdf/default.nix

```diff
@@ -12,11 +12,11 @@ assert enablePrinting -> cups != null;
 
 stdenv.mkDerivation rec {
   pname = "xpdf";
-  version = "4.03";
+  version = "4.04";
 
   src = fetchzip {
     url = "https://dl.xpdfreader.com/xpdf-${version}.tar.gz";
-    sha256 = "09yhvmh1vxjy763nnmawynygp5bh3j4i8ixqja64j11676yl77n6";
+    hash = "sha256-ujH9KDwFRjPIKwdMg79Mab9BfA2HooY5+2PESUgnGDY=";
```

Comment:
None of the CVEs have been solved..?

Reply:
From looking at most of the loop related vulnerabilities surrounding xpdf, there appears to be planning for fixes to these in [xpdf 5](https://forum.xpdfreader.com/viewtopic.php?f=3&t=41263).

Reply:
Ah ok. Nothing to do about it then.